include/dlaf/common/index2d.h

```diff
@@ -83,13 +83,13 @@ class basic_coords {
 
   /// Return a copy of the row or the col index as specified by @p rc.
   template <Coord rc>
-  constexpr IndexT get() const noexcept {
-    if (rc == Coord::Row)
+  IndexT get() const noexcept {
```

Comment:
I think you could keep the `constexpr` here (though I don't know if there's a use case for having it `constexpr`).

Reply:
As discussed separately, it's fine to remove the `constexpr`. It couldn't be used in a `constexpr` context anyway without further changes (the constructor would at least need to be `constexpr`).

Reply:
I removed it because the returned parameter is not built as a constexpr. On the other side, I add it to the `if` because it works on a template parameter.

As discussed offline, it could be potentially be a constexpr function by making constexpr also the constructor. If you agree I would take this change here, and if in the future we introduce more constexpr we can fix this too.